packages/compass-e2e-tests/installers/mac-dmg.ts

```diff
@@ -47,7 +48,9 @@ export async function installMacDMG(
   }
 
   return Promise.resolve({
-    appName,
     appPath: `/Applications/${appName}.app`,
```

Comment:
Could this just be reused?

```suggestion
    appPath: fullDestinationPath,
```

Reply:
Oops yes.